pkg/registry/apis/provisioning/files.go

```diff
@@ -21,6 +21,11 @@ import (
 	"github.com/grafana/grafana/pkg/registry/apis/provisioning/resources"
 )
 
+const (
+	// Files endpoint max size for dashboards etc (5MB)
+	filesMaxBodySize = 5 * 1024 * 1024
```

Comment:
5mb seems reasonable for now -- but long term we may need to tweak it for other kinds of file support.

Dashboards currently have a 15mb limit -- that typically happens when data is saved into the dashbard body.  We aim to migrate these into two objets: 1. the dashboard, 2. the data -- and would still want to support saving them externally